packages/table/preview/index.tsx

```diff
@@ -385,7 +386,7 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
             <label className="pt-label pt-inline tbl-select-label">
                 {label}
                 <div className="pt-select">
-                    <select onChange={this.updateNumberState(stateKey)}>
+                    <select onChange={this.updateNumberState(stateKey)} value={selectedValue as number}>
```

Comment:
do the cast when you define the `const` on line 378, instead of when you want to use it.